hardhat.config.ts

```diff
@@ -55,13 +111,80 @@ const config: HardhatUserConfig = {
       },
     },
   },
-  gasReporter: {
-    enabled: process.env.REPORT_GAS !== undefined,
-    currency: "USD",
+  baseURIs: {
+    // mainnets
+    ethereum: "https://tableland.network/tables/",
```

Comment:
Double checking, do we need to change these URIs to include the chainId?
ref: https://github.com/textileio/go-tableland/blob/2811fbfdefa7005bc4adc3b967920ddb18ee8317/cmd/api/main.go#L133

Reply:
damn, good catch! i think they do need to include chain id